jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/FileUtilTest.java

```diff
@@ -15,116 +15,119 @@
+        () -> assertEquals("..\\..\\", getRelativeFilePath("\\usr\\local\\java\\bin", "\\usr\\local\\")),
+        () -> assertEquals("java\\bin\\java.sh", getRelativeFilePath("\\usr\\local\\", "\\usr\\local\\java\\bin\\java.sh")),
+        () -> assertEquals("..\\..\\..\\", getRelativeFilePath("\\usr\\local\\java\\bin\\java.sh", "\\usr\\local\\")),
+        () -> assertEquals("..\\..\\bin", getRelativeFilePath("\\usr\\local\\", "\\bin")),
+        () -> assertEquals("..\\usr\\local", getRelativeFilePath("\\bin", "\\usr\\local")),
+        () -> assertEquals("", getRelativeFilePath("\\bin", "\\bin"))
+    );
   }
 
     /**
      * Taken from
      * https://github.com/sonatype/plexus-utils/blob/5ba6cfcca911200b5b9d2b313bb939e6d7cbbac6/src/test/java/org/codehaus/plexus/util/PathToolTest.java#L90
      */
     @Test
-    public void testGetRelativeFilePathUnix() {
+    void testGetRelativeFilePathUnix() {
       assumeFalse(isWindows());
-      assertEquals("", getRelativeFilePath(null, null));
-      assertEquals("", getRelativeFilePath(null, "/usr/local/java/bin"));
-      assertEquals("", getRelativeFilePath("/usr/local", null));
-      assertEquals("java/bin", getRelativeFilePath("/usr/local", "/usr/local/java/bin"));
-      assertEquals("java/bin/", getRelativeFilePath("/usr/local", "/usr/local/java/bin/"));
-      assertEquals("../../", getRelativeFilePath("/usr/local/java/bin", "/usr/local/"));
-      assertEquals("java/bin/java.sh",
-        getRelativeFilePath("/usr/local/", "/usr/local/java/bin/java.sh"));
-      assertEquals("../../../", getRelativeFilePath("/usr/local/java/bin/java.sh", "/usr/local/"));
-      assertEquals("../../bin", getRelativeFilePath("/usr/local/", "/bin"));
-      assertEquals("../usr/local", getRelativeFilePath("/bin", "/usr/local"));
-      assertEquals("", getRelativeFilePath("/bin", "/bin"));
+      assertAll(
+              () -> assertEquals("", getRelativeFilePath(null, null)),
+              () -> assertEquals("", getRelativeFilePath(null, "/usr/local/java/bin")),
+              () -> assertEquals("", getRelativeFilePath("/usr/local", null)),
+              () -> assertEquals("java/bin", getRelativeFilePath("/usr/local", "/usr/local/java/bin")),
+              () -> assertEquals("java/bin/", getRelativeFilePath("/usr/local", "/usr/local/java/bin/")),
+              () -> assertEquals("../../", getRelativeFilePath("/usr/local/java/bin", "/usr/local/")),
+              () -> assertEquals("java/bin/java.sh", getRelativeFilePath("/usr/local/", "/usr/local/java/bin/java.sh")),
+              () -> assertEquals("../../../", getRelativeFilePath("/usr/local/java/bin/java.sh", "/usr/local/")),
+              () -> assertEquals("../../bin", getRelativeFilePath("/usr/local/", "/bin")),
+              () -> assertEquals("../usr/local", getRelativeFilePath("/bin", "/usr/local")),
+              () -> assertEquals("", getRelativeFilePath("/bin", "/bin"))
```

Comment:
https://github.com/eclipse/jkube/pull/1621/files#r902266777

Please feel free to refactor junit assertions to assertj